go-controller/pkg/libovsdbops/portgroup.go

```diff
@@ -188,6 +188,31 @@ func AddACLsToPortGroupOps(nbClient libovsdbclient.Client, ops []libovsdb.Operat
 	return m.CreateOrUpdateOps(ops, opModel)
 }
 
+// UpdateACLsOnPortGroupsOps updates the provided ACLs on the provided port group and
+// returns the corresponding ops
+// NOTE: If you pass a slice with 0 ACLs, then the acls will be wiped out
```

Comment:
excellent point! I agree with your comment. I have changed the semantics to not allow for variadic arguments so that the first way of invoking the function is not allowed.
I am ok with changing this to `UpdatePortGroupSetACLsOps`.

Reply:
done.